--- Admin.php.orig	2022-11-12 22:33:05.801000000 -0600
+++ Admin.php.mod	2022-11-12 22:32:05.466000000 -0600
@@ -133,11 +133,15 @@
 			'home_mapping' => [
 				'text' => $this->l10n->t('Attribute to map the users home to.'),
 				'type' => 'line',
 				'required' => true,
 			],
-
+			'user_secret_mapping' => [
+				'text' => $this->l10n->t('Attribute to use as user secret e.g. for the encryption app.'),
+				'type' => 'line',
+				'required' => false,
+			],
 		];
 
 		$firstIdPConfig = isset($providers[0]) ? $this->samlSettings->get($providers[0]['id']) : null;
 		$nameIdFormats = [
 			Constants::NAMEID_EMAIL_ADDRESS => [
